doc/users/plotting/colormaps/grayscale.py

```diff
@@ -23,15 +24,17 @@
 mpl.rcParams['mathtext.sf'] = 'sans'
 mpl.rcParams['mathtext.fallback_to_cm'] = 'True'
 
-# Have colormaps separated into categories: http://matplotlib.org/examples/color/colormaps_reference.html
+# Have colormaps separated into categories:
+# http://matplotlib.org/examples/color/colormaps_reference.html
 
 cmaps = [('Sequential',     ['Blues', 'BuGn', 'BuPu',
                              'GnBu', 'Greens', 'Greys', 'Oranges', 'OrRd',
                              'PuBu', 'PuBuGn', 'PuRd', 'Purples', 'RdPu',
                              'Reds', 'YlGn', 'YlGnBu', 'YlOrBr', 'YlOrRd']),
          ('Sequential (2)', ['afmhot', 'autumn', 'bone', 'cool', 'copper',
-                             'gist_heat', 'gray', 'hot', 'pink',
-                             'spring', 'summer', 'winter']),
+                             'gist_heat', 'gray', 'hot', 'inferno', 'magma',
+                             'pink', 'plasma', 'spring', 'summer', 'viridis',
```

Comment:
Makes sense, Im away until Wednesday but will try to finish it after. Perhaps we should add some more metadata to the colormaps. This info is duplicated 3 or 4 times in various examples.